bazel/external/quiche.BUILD

```diff
@@ -209,6 +210,14 @@ cc_library(
     deps = [":quic_platform"],
 )
 
+cc_library(
+    name = "quic_buffer_allocator_interface_lib",
```

Comment:
How about "quic_buffer_allocator_lib", which is the name used internally?

Reply:
+1 -- most build targets in envoy end with "_lib" or "_interface", not both.